iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js

```diff
@@ -264,6 +268,7 @@ iXBRLViewer.prototype.setProgress = function (msg) {
          * https://bugs.chromium.org/p/chromium/issues/detail?id=675795 
          */
         window.requestAnimationFrame(function () {
+            console.log(`%c [Progress] ${msg} `, 'background: #77d1c8; color: black;');
```

Comment:
Is this console logging intentional?

Reply:
I found it helpful for my own use when learning about the pre-processing pipeline but if we're trying to keep the dev console clean I'm happy to exclude this. 

Maybe a `?dev` flag is something we could consider?